### PR TITLE
secret hide

### PR DESCRIPTION
**Pull request by SumiranAgg**

### CODE REVIEW

File: login/lib/main.js [hunk 1]
```diff
@@ -36,8 +36,13 @@ function main() {
...
                 throw new Error("Not all values are present in the creds object. Ensure clientId, clientSecret, tenantId and subscriptionId are supplied");
             }
-            throwIfError(utility_1.execSync("az", "login --service-principal -u \"" + servicePrincipalId + "\" -p \"" + servicePrincipalKey + "\" --tenant \"" + tenantId + "\""));
-            throwIfError(utility_1.execSync("az", "account set --subscription \"" + subscriptionId + "\""));
+            let option = {
+                silent: true,
+                outStream: process.stdout,
+                errStream: process.stderr
+            };
+            throwIfError(utility_1.execSync("az", "login --service-principal -u \"" + servicePrincipalId + "\" -p \"" + servicePrincipalKey + "\" --tenant \"" + tenantId + "\"", option));
```
Comment (kmkumaran) [inline login/lib/main.js]: What is the experience now for users when the login fails, given that we are passing silent option.

File: login/node_modules/.bin/tsc [hunk 2]
```diff
@@ -1,15 +0,0 @@
-#!/bin/sh
```
Comment (kmkumaran) [inline login/node_modules/.bin/tsc]: Do we need these changes in PR?

File: login/lib/main.js [hunk 3]
```diff
@@ -36,11 +36,17 @@ function main() {
...
+            throwIfError(utility_1.execSync("az", "account set --subscription \"" + subscriptionId + "\"", option));
+            console.log("Login successful.");
         }
         catch (error) {
-            core.debug("Login failed.");
+            console.log("Login failed.");
```
Comment (kmkumaran) [inline login/lib/main.js]: Login failed. Please check the credentials

File: login/lib/main.js (hunk 1 above)
Comment (SumiranAgg) [reply login/lib/main.js]: this is the output 
image
No creds are shown here as well.

File: login/lib/main.js (hunk 3 above)
Comment (SumiranAgg) [reply login/lib/main.js]: Will update

**Comment by TingluoHuang** [comment]
Looks good.

**Pull request closed without merging**